src/engines/kotor/gui/main/main.cpp

```diff
@@ -64,6 +67,19 @@ MainMenu::MainMenu(const Version &gameVersion, Module &module, ::Engines::Consol
 	addBackground(kBackgroundTypeMenu);
 
 	startMainMusic();
+
+	_malakScene.reset(new Graphics::Aurora::SubSceneQuad);
+
+	_malakModel.reset(loadModelObject("mainmenu"));
+	_malakModel->playAnimation("default", true, -1);
```

Comment:
Can you check that _malakModel is != 0 before dereferencing it here? loadModelObject() might return 0 if the model can't be loaded.